crates/iroha/tests/multisig.rs

```diff
@@ -476,7 +489,11 @@ fn multisig_recursion_base(suite: TestSuite) -> Result<()> {
     }
 
     // Check if the multisig transaction has executed
-    let res = test_client.query_single(FindAccountMetadata::new(transaction_target, key.clone()));
+    let res = test_client
+        .query(FindAccounts)
+        .filter_with(|account| account.id.eq(transaction_target))
+        .select_with(|account| account.metadata.key(key.clone()))
+        .execute_single();
```

Comment:
```suggestion
    let res = test_client
        .query_single(FindAccounts::new()
            .filter_with(|account| account.id.eq(transaction_target))
            .select_with(|account| account.metadata.key(key.clone()))
        );
```

this could have been an alternative API. Just comparing it to how we `submit` instructions/transactions